tests/test_survival.py

```diff
@@ -0,0 +1,152 @@
+            },
+            {
+                "_case_id": "39b84ffe-fee5-48aa-b244-6c27aab4248c",
+                "demographic": [{"days_to_death": None, "vital_status": "Alive"}],
+                "diagnoses": [{"days_to_last_follow_up": 734}],
+                "project_id": "MMRF-COMMPASS",
+                "submitter_id": "MMRF_2377",
+            },
+            {
+                "_case_id": "6912c587-3794-440a-ac3f-620869010d93",
+                "demographic": [{"days_to_death": None, "vital_status": "Alive"}],
+                "diagnoses": [{"days_to_last_follow_up": 92}],
+                "project_id": "MMRF-COMMPASS",
+                "submitter_id": "MMRF_1456",
+            },
+        ],
+    }
+}
+
+survival_response = {
+    "results": [
+        {
+            "meta": {"id": 4881527232},
+            "donors": [
+                {
+                    "time": 92,
+                    "id": "6912c587-3794-440a-ac3f-620869010d93",
+                    "submitter_id": "MMRF_1456",
+                    "project_id": "MMRF-COMMPASS",
+                    "survivalEstimate": 1.0,
+                    "censored": True,
+                },
+                {
+                    "time": 734,
+                    "id": "39b84ffe-fee5-48aa-b244-6c27aab4248c",
+                    "submitter_id": "MMRF_2377",
+                    "project_id": "MMRF-COMMPASS",
+                    "survivalEstimate": 1.0,
+                    "censored": True,
+                },
+                {
+                    "time": 769,
+                    "id": "a88a74e1-4a3f-44e9-a521-7ba612cd8935",
+                    "submitter_id": "MMRF_1392",
+                    "project_id": "MMRF-COMMPASS",
+                    "survivalEstimate": 0.7500000000000001,
+                    "censored": False,
+                },
+                {
+                    "time": 876,
+                    "id": "abcbdf03-a7e9-412f-9edd-4ffb93436adb",
+                    "submitter_id": "MMRF_2265",
+                    "project_id": "MMRF-COMMPASS",
+                    "survivalEstimate": 0.7500000000000001,
+                    "censored": True,
+                },
+                {
+                    "time": 1007,
+                    "id": "4e123b99-32aa-4ef2-a709-edeb8b9e48a1",
+                    "submitter_id": "MMRF_1980",
+                    "project_id": "MMRF-COMMPASS",
+                    "survivalEstimate": 0.7500000000000001,
+                    "censored": True,
+                },
+                {
+                    "time": 1467,
+                    "id": "07f4512a-7188-4db1-8571-fd74564dd3f0",
+                    "submitter_id": "MMRF_1325",
+                    "project_id": "MMRF-COMMPASS",
+                    "survivalEstimate": 0.7500000000000001,
+                    "censored": True,
+                },
+            ],
+        }
+    ],
+    "overallStats": {},
+}
+
+
+def mock_guppy_data(app, data):
```

Comment:
we can move `mock_guppy_data` to a `tests/utils.py` file since we need it in both test files